src/github.com/getlantern/flashlight/pac.go

```diff
@@ -41,16 +44,33 @@ func setUpPacTool() error {
 	return nil
 }
 
+func setProxyAddr(addr string) {
```

Comment:
Yes! That sounds right to me.

On Saturday, March 14, 2015, oxtoacart <notifications@github.com> wrote:

> In src/github.com/getlantern/flashlight/pac.go
> <https://github.com/getlantern/flashlight-build/pull/114#discussion_r26438559>
> :
>
> > @@ -41,16 +44,33 @@ func setUpPacTool() error {
> >  	return nil
> >  }
> >
> > +func setProxyAddr(addr string) {
>
> In fact, an interesting project would be an extension-based version of
> detour. In addition to the potential performance benefit, it would also
> allow for doing detection on HTTPS sites.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/getlantern/flashlight-build/pull/114/files#r26438559>.
>


-- 
--
President
Brave New Software Project, Inc.
https://www.getlantern.org
pgp A998 2B6E EF1C 373E 723F A813 045D A255 901A FD89